Makefile

```diff
@@ -1,10 +1,17 @@
-.PHONY: depend up down
+.PHONY: depend set-region up down
 
 depend:
 	ansible-galaxy install -r ./provision/requirements.yml
 
-up: depend
+set-region:
+	export AWS_DEFAULT_REGION=us-west-2
+
+up:
+	make depend
+	make set-region
 	ansible-playbook --tags up ./provision/main.yml
 
-down: depend
+down:
```

Comment:
I've just noticed you removed the `depend` from the dependencies list. Could you clarify why?

Reply:
I added instead `make depend`

Related to the question here - https://github.com/coffr/elk/pull/5/commits/3d4d3e718c7545d1e840c35ae914cd26ed9171dc#r1062320268

Reply:
Adding `make depend` is not the `make` way